src/components/policy/src/policy/include/policy/policy_types.h

```diff
@@ -234,7 +234,8 @@ struct AppPermissions {
           appRevoked(false),
           appPermissionsConsentNeeded(false),
           appUnauthorized(false),
-          requestTypeChanged(false) {
+          requestTypeChanged(false),
```

Comment:
@justinjdickow , invalid order of initialize list. Why you didn`t build your pr?